nerdlets/groundskeeper-nerdlet/index.js

```diff
@@ -544,9 +544,10 @@ export default class Groundskeeper extends React.Component {
     analysis.outdatedTable = {
       columns: [
         {
-          dataField: 'agentAge',
+          dataField: 'agentAge[1]',
           text: 'Agent age',
           sort: true,
+          formatter: (cell, row) => `${cell} weeks old`,
```

Comment:
Thanks for the sorting fix! My only quibble is that `-1 weeks old` is confusuing when we couldn't find the agent's age. Want to put a ternary in here, something like 
```(cell, row) => (cell >= 0) ? `${cell} weeks old` : 'Unknown'```

Reply:
That's a good point. Update coming in a few!